sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -62,6 +62,7 @@ def load(
     key_vault_options: Optional[AzureAppConfigurationKeyVaultOptions] = None,
     refresh_on: Optional[List[Tuple[str, str]]] = None,
     refresh_interval: int = 30,
+    on_refresh_success: Optional[Callable] = None,
```

Comment:
Could we also updtate the preceeding paragraph? it appears we talk about `on_refresh_error` only here. It would be good to add information on `on_refresh_success` too in the write-up